config/application.rb

```diff
@@ -6,6 +6,8 @@
 require_relative '../lib/spree/core/environment'
 require_relative '../lib/spree/core/mail_interceptor'
 
+require "view_component/storybook/engine"
```

Comment:
it's almost the same question that we had already: do we need Storybook in production? I guess somedays, yes (at least why not in certain specific cases), but at this time, I think it's more a developper tools so I will make it only for development, and re-create a PR if we want to push StoryBook in production.

Reply:
It's now in the `config/environments/development.rb` file: 

https://github.com/openfoodfoundation/openfoodnetwork/pull/7284/files#diff-d3c4b3f41072daa416f1920511e9b2e26caea8c5cec0a14cb9508589a4dafa47R1-R2

Reply:
I guess I didn't understand your point, because if I put this one under `config/environments/development.rb` got the error:

```
uninitialized constant ViewComponent::Storybook::StoriesController (NameError)
```

Reverted, and put into `config/application.rb`

Reply:
I agree with @mkllnk, it feels like we probably shouldn't be loading this engine in `application.rb`.

I'll take a look at it now, and see what we can do :ok_hand:

Reply:
Looks like it's time to `bundle install && yarn install` and RTFM :sweat_smile: